lib/node_modules/@stdlib/blas/ext/base/dapxsumpw/manifest.json

```diff
@@ -34,7 +34,12 @@
 				"-lm"
```

Comment:
Indentation should be changed to two spaces here.

Would you please also update the file to have separate configurations for benchmarks and examples. The linked changes in the RFC or merged PRs should give guidance on how the file has to be updated.